database/tools/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+
+# Generated by TypeDB Cargo sync tool.
+# Do not modify this file.
+
+features = {}
+
+[package]
+	name = "read-wal"
+	edition = "2021"
+	version = "0.0.0"
+
+[dependencies]
+
+	[dependencies.clap]
+		features = ["color", "default", "derive", "error-context", "help", "std", "suggestions", "usage", "wrap_help"]
+		version = "4.5.24"
+		default-features = false
+
+	[dependencies.durability]
+		path = "../../durability"
+		features = []
+		default-features = false
+
+	[dependencies.concept]
+		path = "../../concept"
+		features = []
+		default-features = false
+
+	[dependencies.storage]
+		path = "../../storage"
+		features = []
+		default-features = false
+
+[[bin]]
+	path = "read-wal.rs"
+	name = "read-wal"
+
+[[bin]]
+	path = "replay-wal.rs"
+	name = "replay-wal"
+
```

Comment:
Note: Cargo sync tool can't actually handle two binaries in the same package! This entry was added manually.